strings/hint.txt

```diff
@@ -132,3 +132,7 @@ Pugs are the
 superior dog
 breed.  >
 -
+You are
+playing
+ALTTPR.     >
```

Comment:
I believe `>` ends up being 2 of the 14 chars on a line, so I think this line might mess up.

Reply:
This is on the 13th spot, and it is at the same position as some other hints.  It should work.